### DISCUSSION TITLE
Disabling Named Aliasing

### DISCUSSION THREAD

**Discussion by berchn** [tree-sitter/tree-sitter#1077] [Q&A] [answered]
Is it possible to disable named aliases from appearing in the parse tree opting for their content instead? I'm currently using the `src/grammar.json` and `src/node-types.json` of a parser to generate code that can reassemble the source text of what has been parsed. This is done by matching the state of a node with its relevant rule. In several cases, the node's state does not match what the rule specifies. This is exclusively a problem caused by named aliases.

**Comment by ObserverOfTime** [comment, marked as answer]
Don't use aliases?
